ipatests/test_integration/test_ntp_options.py

```diff
@@ -175,6 +181,13 @@ def test_replica_promotion_with_ntp_options(self):
         tasks.install_master(self.master, setup_dns=False)
         tasks.install_client(self.master, self.replica)
 
+        # Testing command line options that may not be used together, there
+        # is theoretical no need to configure the firewall in this case,
+        # because the installation will not succeed. But it has been added
+        # to make sure that the test will not fail because of the missing
+        # firewall configuration if the options will not conflict.
+        Firewall(self.replica).enable_services(["freeipa-ldap",
+                                                "freeipa-ldaps"])
```

Comment:
So in other words you want the test to fail because of the conflicting options and not because the firewall wasn't configured?

Reply:
Yes, just in the case the conflict is not there any more.. then the test would fail because of the missing firewall configuration. But the test should fail because the replica was installed successfully by ignoring the conflict.